### ISSUE TITLE
[QUESTION] How do you add and remove blocks using Search?

### ISSUE THREAD

**Issue by going2021** [bebeli555/CookieClient#19]
The in game description is that you have to do search add block and search remove block but I have multiple questions about how you do that. Is it in in game chat or somewhere else and do you put a . before the command or a /, I was also wondering if blocks with two words in the name have an underscore between the words.

I could not find anywhere online where someone describes how to use this so I just wanted clarification.

**Comment by bebeli555** [comment]
you need to send it in the chat as a command so you put the prefix which is ++ in default then search add and then the block ID. you can find the block ids from websites just search like minecraft 1.12.2 block ids on google. like "++search add 1" would work
